lib/FieldType/Mapper/RichTextFormMapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\Core\FieldType\RichText\Type;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class RichTextFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'numRows',
+                'integer',
+                [
+                    'empty_data' => 10,
+                    'required' => true,
+                    'property_path' => 'fieldSettings[numRows]',
+                    'label' => 'field_definition.ezrichtext.num_rows',
+                ]
+            )
+            ->add(
+                'tagPreset',
```

Comment:
@rolandbenedetti Ok if we skip this one for now? as @dpobel says we probably want to do this more inline with what is planned for UI here and also long time issue is that this should rather be permission handled\* if possible 


\* *However that opens a range of new issues again, like user editing existing RichText with tags he does not have access to add himself, how to handle that.. so maybe not.*